src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -213,7 +275,10 @@ DomainList.propTypes = {
   DefaultImage: PropTypes.element,
   isCommunity: PropTypes.bool,
   handleSetFollow: PropTypes.func,
-  handleSetUnFollow: PropTypes.func
+  handleSetUnFollow: PropTypes.func,
+  isBlockedSection: PropTypes.bool,
+  handleSetBlock: PropTypes.func,
+  handleSetUnblock: PropTypes.func
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New prop types have been defined. It's good practice to ensure that these props are always provided to prevent unexpected behavior. If they are optional, consider providing default prop values.


<!-- This is an auto-generated comment by OSS CodeRabbit -->